commands-yml/validator.js

```diff
@@ -39,7 +39,7 @@ export default {
   'client_docs.java': {url: true},
   'client_docs.javascript_wdio': {url: true},
   'client_docs.javascript_wd': {url: true},
-  'client_docs.ruby': {url: true},
+  'client_docs.ruby': {},
```

Comment:
Curren spec doesn't allow input multiple URLs in the same place.
To achieve https://github.com/appium/appium/pull/11025/files#diff-3be2944a88736b0d35e8c71a33159dd4R31 , I relaxed the validation for Ruby as temporary.